python/python/glide/async_commands/cluster_commands.py

```diff
@@ -514,3 +514,42 @@ async def flushall(
             TClusterResponse[TOK],
             await self._execute_command(RequestType.FlushAll, args, route),
         )
+
+    async def lolwut(
+        self,
+        version: Optional[int] = None,
+        parameters: Optional[List[int]] = None,
+        route: Optional[Route] = None,
+    ) -> str:
+        """
+        Displays a piece of generative computer art and the Redis version.
+
+        See https://valkey.io/commands/lolwut for more details.
+
+        Args:
+            version (Optional(int)): Version of computer art to generate.
+            parameters (Optional[str]): Additional set of arguments in order to change the output:
+                For version <code>5</code>, those are length of the line, number of squares per row, and number of squares per column.
+                For version <code>6</code>, those are number of columns and number of lines.
+            route (Optional[Route]): The command will be routed to all primary nodes, unless `route` is provided,
```

Comment:
isn't it routed to a random node instead of all primaries?
```suggestion
            route (Optional[Route]): The command will be routed to a random node, unless `route` is provided,
```